normalized-cache/src/commonMain/kotlin/com/apollographql/cache/normalized/SimpleApolloStore.kt

```diff
@@ -20,8 +20,8 @@ import kotlin.reflect.KClass
  * A wrapper around [ApolloStore] that provides a simplified API for reading and writing data.
  */
 class SimpleApolloStore(
```

Comment:
Not directly related to this PR but I would probably do the opposite naming choice. Keep `ApolloStore` for the thing that is used the most (this) and use something else for `ApolloStore`. Maybe even possibly hide it from public API altogether?

Reply:
Hmm we still need both the wrapper and the “wrapped” to be public because setting it takes the wrapped but getting returns the wrapper

```kotlin
// setting
fun ApolloClient.Builder.store(store: ApolloStore, writeToCacheAsynchronously: Boolean = false): ApolloClient.Builder

// getting
val ApolloClient.store: SimpleApolloStore
```

Re: naming, we could do `ApolloStore` → `IApolloStore` and `SimpleApolloStore` → `ApolloStore`  but not super fan of `I` and it’s different from what we do elsewhere.

Maybe at least `SimpleApolloStore` should be called something else (but not sure what!), since it’s not an `ApolloStore` implem.

Reply:
Setting it doesn't really need a store, does it? I suspect most of the users use the `.normalizedCache()` function, and not `.store()`?

The only use case for using `.store()` is if someone wants to use their own `ApolloStore` implementation but I'm not sure what that would be useful for....

Reply:
>  I suspect most of the users use the .normalizedCache() function, and not .store()?

Most probably yes (that's what we show in the docs).

Also not sure of specific use cases to bring your own implementation (but IIRC Netflix does it  😅)  - but I think it's a nice flexibility to be able to do it and would be a bit sad to remove it IMO.

Reply:
As discussed, `.apolloStore` now returns `ApolloStore`, which is the wrapper. The interface is now called `CacheManager`.